"master" branch is broken (TypeError: __init__() got an unexpected keyword argument 'metavar')

(Part of) commit 9e66a2e (that came from https://github.com/truckersmp-cli/truckersmp-cli/pull/167/commits/fe64c80f3b37390cdae170c57dc8ce024c4f991f and https://github.com/truckersmp-cli/truckersmp-cli/pull/167/commits/853f880c2c13e81f1864b686f83545608c779649) broke everything:

```
$ git clone https://github.com/truckersmp-cli/truckersmp-cli.git
$ cd truckersmp-cli
$ ./truckersmp-cli -h
Traceback (most recent call last):
  File "/path/to/workdir/truckersmp-cli/./truckersmp-cli", line 19, in <module>
    main()
  File "/path/to/workdir/truckersmp-cli/truckersmp_cli/main.py", line 82, in main
    arg_parser = create_arg_parser()[0]
  File "/path/to/workdir/truckersmp-cli/truckersmp_cli/args.py", line 309, in create_arg_parser
    store_actions.append(parser.add_argument(
  File "/usr/lib/python3.9/argparse.py", line 1428, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'metavar'

$ ./gen_completions -b /path/to/bash_completion -z /path/to/zsh_completion
Traceback (most recent call last):
  File "/path/to/workdir/truckersmp-cli/./gen_completions", line 216, in <module>
    sys.exit(main())
  File "/path/to/workdir/truckersmp-cli/./gen_completions", line 169, in main
    for act in create_arg_parser()[1]:
  File "/path/to/workdir/truckersmp-cli/truckersmp_cli/args.py", line 309, in create_arg_parser
    store_actions.append(parser.add_argument(
  File "/usr/lib/python3.9/argparse.py", line 1428, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'metavar'

$ LANG=C make
./gen_completions --zsh-completion _truckersmp-cli
Traceback (most recent call last):
  File "/path/to/workdir/truckersmp-cli/./gen_completions", line 216, in <module>
    sys.exit(main())
  File "/path/to/workdir/truckersmp-cli/./gen_completions", line 169, in main
    for act in create_arg_parser()[1]:
  File "/path/to/workdir/truckersmp-cli/truckersmp_cli/args.py", line 309, in create_arg_parser
    store_actions.append(parser.add_argument(
  File "/usr/lib/python3.9/argparse.py", line 1428, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'metavar'
make: *** [Makefile:9: _truckersmp-cli] Error 1
```

--- a/.github/workflows/build_and_upload_inject_program.yaml
+++ b/.github/workflows/build_and_upload_inject_program.yaml
@@ -29,7 +29,7 @@ jobs:
       - name: Build truckersmp-cli.exe
              # we need "make clean" because we already have truckersmp-cli.exe
              # and "make" does nothing even when truckersmp-cli.c is changed
-        run: make clean && make truckersmp_cli/truckersmp-cli.exe
+        run: make clean && make
       - name: Strip truckersmp-cli.exe
         run: x86_64-w64-mingw32-strip truckersmp_cli/truckersmp-cli.exe
       - uses: actions/upload-artifact@v1

--- a/truckersmp_cli/args.py
+++ b/truckersmp_cli/args.py
@@ -311,8 +311,7 @@ SteamCMD can use your saved credentials for convenience.
         default="-nointro -64bit",
         help="""specify ATS/ETS2 options
                 Note: If specifying one option, use "--game-options=-option" format
-                [Default: "-nointro -64bit"]""",
-        action="store_true"))
+                [Default: "-nointro -64bit"]"""))
     store_actions.append(parser.add_argument(
         "--native-steam-dir", metavar="DIR", type=str,
         default="auto",

